src/smd_strings.r2

```diff
@@ -0,0 +1,412 @@
+"wz SEGA GENESIS    (C)SEGA 1995.MARCART. BRADESCO                                  CART. BRADESCO                                  GM 00000001-00W`J               " @ 256
```

Comment:
i think moving files like this to bins will make it harder to debug and fix bugs in the flag filtering and command parsing. it is better to keep the scripts in r2 repo, and just use the bins/src as a place to keep the source of the programs in there.

Reply:
I was going through the files to see which ones I should move back to the r2 repository. For a few files, I am not sure about whether I should move them back to the r2 repo or leave them here. I guess I should move `src/smd_strings.r2`, `src/gdb-reg-profile-invalid.txt`, `src/gdb-reg-profile.txt`, `src/palette.r2` back to the r2 repo. Is that right?

Reply:
Would it be right to leave `src/r2test.spp `, `src/test-webserver.sh` and `src/test-webserver.bat` over here in bins/src? Also, I haven't exactly understood what `src/no-nl-at-eof.r2` and `src/keyword` do.

Reply:
Yes, those files that depend on r2 commands must go into the r2 repo, the bins repo should only contain binaries. Another issue i have noticed is that projects should be moved too.

Reply:
I think I have understood which scripts to move back to the r2 repo. However, I haven't understood which projects you are referring to that need to be moved.

Reply:
What do I do if I am not sure whether a particular file depends on r2 commands? Should I leave them here for the time being?